benchmarking/smartseq2/merge_expression_matrix/merge_table_quants.py

```diff
@@ -0,0 +1,93 @@
+import argparse
```

Comment:
As @ambrosejcarr already mentioned before, it would be great to run `pycodestyle` aginst the python scripts. e.g. 
`pip install pycodestyle` and then `pycodestyle merge_table_quants.py`